tools/wdio/plugins/jil-commands.mjs

```diff
@@ -4,8 +4,12 @@
 export default class TestingServerWorker {
   async before (capabilities, context, browser) {
     browser.addCommand('waitForFeature', async function (feature) {
+      const command = `window.NREUM && window.NREUM.activatedFeatures && window.NREUM.activatedFeatures['${feature}']`
       await browser.waitUntil(
-        () => browser.execute((feat) => `window.NREUM && window.NREUM.activatedFeatures && window.NREUM.activatedFeatures['${feat}']`, feature),
+        // () => browser.execute((feat) => `window.NREUM && window.NREUM.activatedFeatures && window.NREUM.activatedFeatures['${feat}']`, feature),
+        () => browser.execute(function (feat) {
```

Comment:
Does `browser.execute(...)` need to get transpiled, or do we need to enforce ES5 syntax?

Reply:
WDIO does not transpile the function code automatically. We could override the `execute` command to add transpiling. If we want to do that, I can add that to this PR.

Reply:
Looking through the code of wdio, I don't think its a good idea to override the `execute` command. I think we should just adhere to ES5 syntax to support IE. Or we could create a new command that assumes a function being passed in and babelifies it.

https://github.com/webdriverio/webdriverio/blob/main/packages/webdriverio/src/commands/browser/execute.ts
https://github.com/webdriverio/webdriverio/blob/main/packages/devtools/src/commands/executeScript.ts